static/locales/fr/translation.json

```diff
@@ -7,26 +7,26 @@
 			"title": "Paramètres du lobby",
 			"gamehostonly": "Seul l'hôte de la partie peut modifier cela!",
 			"inlobbyonly": "Vous ne pouvez changer cela que dans le lobby!",
-			"voicedistance": "Distance vocal",
-			"voicedistance_impostor": "Distance vocal imposteurs",
+			"voicedistance": "Distance vocale",
+			"voicedistance_impostor": "Distance vocale imposteurs",
 			"wallsblockaudio": "Les murs bloquent l'audio",
-			"visiononly": "Entendre uniquement les gens que l'on voit",
+			"visiononly": "Entendre uniquement les gens visibles",
 			"impostorshearsghost": "Les imposteurs entendent les fantômes",
 			"hear_imposters_invents": "Entendre les imposteurs dans les vents",
 			"private_talk_invents": "Discussion privée dans les vents",
 			"comms_sabotage_audio": "Saboter les communications désactive le vocal",
 			"hear_through_cameras": "Entendre à travers les caméras",
 			"ghost_only": "Seuls les fantômes peuvent parler/entendre",
 			"ghost_only_warning": "Cette option désactive le vocal pour les joueurs vivants.",
-			"ghost_only_warning2": "Seuls les fantômes peuvent parler activé.",
+			"ghost_only_warning2": "Seuls les fantômes peuvent parler.",
 			"meetings_only": "Meetings/Lobby uniquement",
 			"meetings_only_warning": "Cela désactive le son pour les joueurs vivants en dehors des meetings.",
-			"meetings_only_warning2": "Meetings/Lobby uniquement activé.",
+			"meetings_only_warning2": "Vous ne pouvez que parler dans les Meetings/le Lobby.",
```

Comment:
Again, meant to be a warning not simply restating the setting. Context is important with these messages.